meta/client.go

```diff
@@ -0,0 +1,169 @@
+package meta
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/XiaoMi/pegasus-go-client/session"
+	"github.com/bluele/gcache"
+	"github.com/samuel/go-zookeeper/zk"
+)
+
+// TODO(jiashuo) store config file
```

Comment:
https://github.com/spf13/viper + yaml

Reply:
Ok,`config file` and `log file` is expected delay at next PR